src/sentry/integrations/jira/integration.py

```diff
@@ -139,6 +139,28 @@ def get_project_config(self):
 
         return configuration
 
+    def update_organization_config(self, data):
+        """
+        Update the configuration field for an organization integration.
+        """
+        config = self.org_integration.config
+
+        if 'sync_status_forward' in data:
+            project_mappings = data.pop('sync_status_forward')
+
+            data['sync_status_forward'] = bool(project_mappings)
+
+            for project_id, statuses in project_mappings.items():
+                IntegrationExternalProject.objects.create(
```

Comment:
What about removing them?

Reply:
A test for removal i.e. deleting `IntegrationExternalProjects` that have been removed from the UI  and checking that `'sync_status_forward'` is set to `False` if all of the project mappings are removed would be useful here.

Reply:
Also don't forget about just updating pre-existing `IntegrationExternalProjects`